src/app/Plans/components/Wizard/PlanWizard.tsx

```diff
@@ -412,7 +412,7 @@ const PlanWizard: React.FunctionComponent = () => {
         <>
           <Prompt
             when={forms.isSomeFormDirty && mutationStatus === 'idle'}
-            message="You have unsaved changes, are you sure you want to leave this page?"
+            message="Leave this page? Your unsaved changes will be lost."
```

Comment:
I'd replace "your" with "all" so that users understand that any of their changes are at risk of being lost, unless they save them:

Leave this page? All unsaved changes will be lost.